index.js

```diff
@@ -0,0 +1,24 @@
+"use strict"
+
+// The list of rules that can be used in Solhint can be found at:
+// https://protofire.github.io/solhint/docs/rules.html
+
+// The rules below uses the recommended rules from solhint with some
+// modifications
+module.exports = {
+  "extends": "solhint:recommended",
+  'rules': {
+    "imports-on-top": "error",                     // recommended uses warn
+    "use-forbidden-name": "error",                 // recommended uses warn
+    "no-unused-vars": "error",                     // recommended uses warn
+    "reason-string": ["error", {"maxLength":250}], // recommended uses warn
+    "max-line-length": ["error", 145],             // not present in recommended
```

Comment:
Why 145 and not 80?

Reply:
The `solium` version used `max-len` with a default value, which was 145, but I can change it to 80.

Reply:
After thinking more about it, I'd drop this line completely and left the line length to `prettier` that will be invoked right after `solhint` in `lint:sol` and `lint:fix:sol` commands.

Reply:
Ok, I removed the `max-line-length` check completely.